nibabel/gifti/gifti.py

```diff
@@ -192,10 +179,18 @@ def data_tag(dataarray, encoding, datatype, ordering):
         raise NotImplementedError("In what format are the external files?")
     else:
         da = ''
-    return "<Data>" + da + "</Data>\n"
+
+    data = xml.Element('Data')
+    data.text = da
+    return data
```

Comment:
Yeah, I read the docs and had tried this. On my machine it set an attribute: `<Data text='blah' />`.